app/core/src/main/java/com/fsck/k9/AccountPreferenceSerializer.kt

```diff
@@ -597,7 +598,7 @@ class AccountPreferenceSerializer(
             maxPushFolders = 10
             isSubscribedFoldersOnly = false
             maximumPolledMessageAge = -1
-            maximumAutoDownloadMessageSize = 32768
+            maximumAutoDownloadMessageSize = 131072
```

Comment:
Consider using a constant for the default value of `maximumAutoDownloadMessageSize`. Then there is no need to suppress `MagicNumber`.